src/Symfony/Component/Finder/Finder.php

```diff
@@ -614,6 +614,20 @@ public function append($iterator)
     }
 
     /**
+     * Check if the any results were found.
+     *
+     * @return bool
+     */
+    public function found()
+    {
+        foreach ($this->getIterator() as $null) {
```

Comment:
might as well update `getIterator()` to `@return \Traversable|SplFileInfo[]` then, to indicate why we add this method. Currently we could rely on `getIterator()->valid()` i believe :)